### PR TITLE
Process Report

### PR DESCRIPTION
**Pull request by jchon11**
Editing my page was actually a lot easier than I thought it would be. Once I learned how to fork and then pull up the page in RStudio, it became a lot like simply modifying text documents. I think the most helpful thing I found was a Github guide on Mastering Markdown. 
(https://guides.github.com/features/mastering-markdown/) 
It gave me all the information I needed to know to do the homework tasks (How to create emphasis, lists, headers and even images). Also, the video on the homework page was really good, and walked me through each step of the process. 
However, I did have one problem, and that was with trying to embed an image. Because I was a bit confused on what would be necessary to make the image actually appear rather than a link, I had to go through a couple different edits to the code. However, I managed to get it displayed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+
+![](https://github.com/jchon11/myrepo/blob/master/joshchon.png)
```
Comment (deblnia) [inline README.md:3]: Good use of a link to get images! You can also download an image, put it in the directory you're Rmd is in, and use a relative path to render. See [here](https://www.earthdatascience.org/courses/earth-analytics/document-your-science/add-images-to-rmarkdown-report/) for more.

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+1. Hiking
+2. Camping
+3. Biking
+4. Watching movies. My favorite movies are...
+  * *The Godfather*
+  * *Citizen Kane*
+  * *The Grand Budapest Hotel*
+  
+### Link
+https://apexlab.uchicago.edu/
```
Comment (deblnia) [inline README.md:23]: To link inline use `[TEXT](link.com)`. (Inline means it renders [like this](https://apexlab.uchicago.edu/))

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X        |              |            |
| **Coding strategy**         |  X         |              |            |
| **Presentation**            |           |     X         |            |
| **Achievement, creativity** |       X    |              |            |
| **Reproducibility**         |      X     |              |            |

Remarks:

* Overall, really good work! 
* I docked presentation points for that URL -- it's generally good practice to en-linken text for presentation purposes. You can also use the keyboard shortcut CNTRL/ CMD + K with a link on your keyboard to make that even easier.

**Comment by oneforae** [comment]
I think this looks great and provides a great description of your progress through the project. GREAT movie picks! I also understand the difficulty embedding the image, but I'm glad you got it all sorted out! I too felt GitHub's description of Markdown was very helpful.

**Comment by dvalverde123** [comment]
This looks really clean! You did a great job with adding an image, using the bold/italics, and adding lists. I liked how ordered and organized it was. There wasn't anything wrong with your assignment, but something interesting to learn for the future might be how to make text different colors! I saw it in another of the assignments I reviewed and I think it would have been a really great addition to your document.

**Pull request closed without merging**